php-auth-system/controllers/AuthController.php

```
@@ -0,0 +1,407 @@
+            $this->user->logout();
+            
+            setFlashMessage('success', 'You have been logged out successfully.');
+        }
+        
+        redirect('/login');
+    }
+    
+    /**
+     * Set remember me cookie
+     */
+    private function setRememberMeCookie($userId) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$userId'. (Unused Code Rules)

(UnusedFormalParameter)